Flutter Open Source Apps/Advanced/readme.md

```diff
@@ -1,6 +1,25 @@
 
 # Advanced Projects Designs
 
-Here we upload the best designs for some Advanced projects.
-Try to upload apps for the ones with advanced knowledge.
-Happy Development !!
\ No newline at end of file
+## Project 1. UNI-Notes: 
+### It not an another notes sharing app instead it have some powerfull feature which make the app stand out. MyNotes is a media sharing app for student where student can share assignment, notes, tutorial, notice, etc in any format with fellow students publically specific to their institute or in a private chat.
+### If any students uploads a media content then all the students in a institute will get notified this we have done using firebase cloud messaging service.
+### Students can chat or share any other media in private chat room, and they will get realtime send, read, delivered receipt.
+
+## Participating the Hacktoberfest?
+### Did I say the hacktoberfest, yes you heard me right it is participating the hacktoberfest 2022
+### To support the Hacktoberfest event, the hacktoberfest label will be added to every pull request made on 1 to 31 October 2022
+### Let's prepare yourself!
+
+## Why do i need to work this?
+### If you reach till here then I am pretty sure you have this question is in your mind why should I work on this project well, if you are flutter developer then this is perfect project for you, you will get to learn alot things about flutter like Bloc, state management, No internet activity, shimmer loading animation, flutter responsiveness, alot of different types of pakages and what not some some additional things like Firebase, github CI/CD pipeline, google auth.
+
+## can I contribute?
+### Yes bcuz why not, you can contribute through code and non code, one thing I can make does't matter if you're a naive or a master
+### Repo link : https://github.com/nitin-787/mynotes
```

Comment:
The Repo link should be:-https://github.com/HemantPra389/Flutter-Open-Source-Apps

Reply:
Can elaborate it more?

Reply:
> Can you elaborate it more?

> If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗 Happy Hacktoberfest2022 !! 🥳



> If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗 Happy Hacktoberfest2022 !! 🥳

Hey Sorry I if miss understood but I owner of this repo https://github.com/nitin-787/Flutter-Open-Source-Apps/tree/nitin/Flutter%20Open%20Source%20Apps/Advanced#readme and I thought this repo that contain different type of flutter projects and I just added mine

Reply:
Do I need to add my whole project here?

Reply:
Yes so that people can find opensource flutter apps at one go